lib/api/core/plugins/pluginsManager.js

```diff
@@ -33,15 +33,35 @@ const
   CircularList = require('easy-circular-list'),
   fs = require('fs'),
   pm2 = require('pm2'),
+  clc = require('cli-color'),
+  semver = require('semver'),
   {
     KuzzleError,
     GatewayTimeoutError,
     PluginImplementationError,
     UnauthorizedError
   } = require('kuzzle-common-objects').errors;
 
+// This is default manifest for all plugin, can be overriden by "manifest.json" file at root path of each plugin
+const DefaultManifest = {
+  name: undefined,
+  version: '1.0.0',
+  threadable: true,
```

Comment:
This comment is outside of the scope of this PR, but IMO, we should eventually remove threaded plugins from Kuzzle. 

This behaviour adds some extra pain and some dependency with pm2.
If a plugin wants to create subprocess, it can perfectly handle it by itself.